back-end/helpers/test-helpers/testIntentGenerator.js

```diff
@@ -0,0 +1,74 @@
+export default class testIntentGenerator {
+    generateOneIntentModel() {
```

Comment:
```suggestion
    generateIntentModel(question, children) {
    return {question, children, total_children, project_id, getPercentages}
}
```
(marking the above as pseudo-code)